doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+* _Show system tray icon:_ If set, qTox will show its icon in your system tray.
+    * _Start in tray:_ On start, qTox will only show its tray icon and no window.
+    * _Minimize to tray:_ The minimize button on the top right, will minimize qTox to its tray icon. There won't be a taskbar item.
+
+* _Auto away after (0 to disable):_ After the specified amount of time, qTox will set your Status to "Away". A setting of 0 will never change your status.
+* _Default directory to save files:_ You can set the directory where qTox puts files you recieved here.
+* _Autoaccept files:_ If set, qTox will automatically accept filetransfers and put them in the directory specified above.
+
+#### Chat
+* _Play sound:_ If checked, qTox will play a sound when you get a new message.
+* _Open window:_ If checked, the qTox window will be opened. <!-- TODO: Will a NEW window be opened? What happens in Multi Window mode? -->
+    * _Focus window:_ If checked, the qTox window will additionally be focused.
+* _Show contacts' status changes:_ ???<!-- TODO: what does this even do?-->
+* _Group chats always notify:_ ???<!-- TODO: what does this even do?-->
+* _Place groupchats at top of friend list:_ If checked, your groupchats will be at the top of the contacts list. <!-- TODO: else what?-->
+* _Faux offline messaging:_ If enabled, qTox will attempt to send messages when a currently offline contact comes online again.<!-- TODO:Is this even correct?-->
+* _Compact contact list:_ If set, qTox will use a contact list layout which takes up less screen space.
+* _Multiple windows mode:_ If enabled, the qTox user interface will be split into multiple independent windows. For details see TODO.<!-- TODO:write detailed explainaiton of multi window mode-->
+    * _Open each chat in an individual window:_ If checked, a new window will be opened for every chat you open.
+
+#### Theme
+* _Use emoticons:_ If enabled, qTox will replace simleys ( e.g. :-) )  with Unicode emoticons.
+* _Smiley Pack:_ You can choose from different sets of shiped emoticon styles. You can also install your own, see ??? for details<!-- TODO: add instructions for adding own smiley packs-->
+* _Emoticon size:_ You can change the size of the emoticons here.
+* _Style:_ Changes the appearance of qTox.<!-- TODO: possible to add own styles? -->
+* _Theme color:_ Changes the colors qTox uses.<!-- TODO: possible to add own color schemes? -->
+* _Timestamp format:_ Change the format in which qTox displays message timestamps.
+* _Date format:_ Same as above for the date.
+
+#### Connection Settings
+* _Enable IPv6 (recommended):_ If enabled, qTox will use IPv4 and IPv6 protocols, whichever is available. If disabled, qTox will only use IPv4.
+* _Enable UDP (recommended):_ If enabled, qTox will use TCP and UDP protocols. If disabled, qTox will only use TCP.<!-- TODO: correct? what advantages/disadvantages does this have? -->
```

Comment:
Some routers have been proven to crash if UDP is enabled. Same goes for IPv6 btw.

Using only TCP is supposed to lower amount of network traffic, although it puts more burden on other network participants, it's generally slower (latency) and has lower throughput (bandwidth), given that network traffic to/from user is being relayed by relay nodes.